web/src/components/data-fields/address/UnitesStatesAddress.tsx

```diff
@@ -22,7 +23,11 @@ export const UnitesStatesAddress = (props: Props): ReactElement => {
 
   return (
     <>
-      <AddressLines1And2 onValidation={props.onValidation} isFullWidth={props.isFullWidth} />
+      <AddressLines1And2
```

Comment:
error handling is not firing for any address components when im tabbint through and all fields are emtpy